x/ibc/keepers.go

```diff
@@ -0,0 +1,9 @@
+package ibc
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// expected bank keeper
+type BankKeeper interface {
+	AddCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) (sdk.Coins, sdk.Tags, sdk.Error)
```

Comment:
Also like having the only import here being the sdk!!!

Reply:
:) me too